docs/webhooks/creating-and-configuring-your-webhook.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Creating & configuring your webhook
+excerpt: Learn how to configure your webhook and how to receive your first delivery
+category: TODO
+slug: webhooks-setup
+---
+
+Now that we understand [what webhooks are and how they work](doc:webhooks), we can create our own
```

Comment:
add a shortened intro from the initial `webhooks` page here, instead of linking off to it